File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.Skins;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        Random r = new Random();
        private void timer1_Elapsed(object sender, System.Timers.ElapsedEventArgs e)
        {
            defaultLookAndFeel1.LookAndFeel.SkinName = SkinManager.Default.Skins[r.Next(SkinManager.Default.Skins.Count)].SkinName;
        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Skins

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private r As New Random()
		Private Sub timer1_Elapsed(ByVal sender As Object, ByVal e As System.Timers.ElapsedEventArgs) Handles timer1.Elapsed
			defaultLookAndFeel1.LookAndFeel.SkinName = SkinManager.Default.Skins(r.Next(SkinManager.Default.Skins.Count)).SkinName
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load

		End Sub
	End Class
End Namespace